Architecture_Engine/Compute/CeilingTiles.cs

```diff
@@ -49,6 +49,12 @@ public static partial class Compute
         [Output("ceilingTiles", "Closed Ceiling Tile regions contained within the Ceiling.")]
         public static List<CeilingTile> CeilingTiles(Ceiling ceiling, List<Line> ceilingTileLines, double angleTolerance = BH.oM.Geometry.Tolerance.Angle, double distanceTolerance = BH.oM.Geometry.Tolerance.Distance, int decimalPlaces = 6)
         {
+            if(ceiling == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Cannot compute the ceiling tiles for a null ceiling.");
+                return null;
```

Comment:
Why null and not an empty list?
```suggestion
                return new List<CeilingTile>();
```